content/docs/guides/transforms/scrape-data-from-a-website.md

```diff
@@ -0,0 +1,129 @@
+res = http.get(sitetoscrape).body()
+```
+
+### Step 3: Extract text from specific elements
+
+With this code, we will use the Starlark html package to find the elements containing the data we want to scrape.  In this example, each street closure consists of a `<strong>` element with the streets to be closed, followed by a `<p>` element with a description of the closure.
+
+![screenshot of NYC DOT weekend traffic advisory website](/img/docs/guides/transforms/scrape-data-from-a-website/nycdot-weekend-traffic-advisory.png)
+
+
+![screenshot of NYC DOT weekend traffic advisory website html source](/img/docs/guides/transforms/scrape-data-from-a-website/nycdot-weekend-traffic-advisory-html.png)
+
+From inspecting the HTML of the site, we can see that all of the `<strong>` and `<p>` elements of interest are sibling elements under a `<div>` with an `id` of 'content'.  Once the `children()` of this `<div>` have been selected, we can iterate over them to find each `<strong>` that is immediately followed by `<p>` and extract the text from these elements.
+
+For good measure, we can also grab the borough from the `<h2>` elements found on the page, and attach them to each street closure.
+
+```python
+# use html() to parse the raw html
+root = html(res)
+# get all child nodes of the div with id="content"
+content = root.find('#content').children()
+
+# create an empty list with a list as header row
+rows = [['borough', 'streets_closed', 'description']]
+# when we encounter an h2, we will set currentBoro so we can add a boro to each event row
+currentBoro = ''
+
+# iterate over the children of #content
+for i in range(0,content.len()):
+    # set up variables for the current node and the next node
+    currentElement = content.eq(i)
+    nextElement = content.eq(i+1)
+
+    # check for h2, if true, get the node's text and set currentBoro
+    if currentElement.is_selector('h2'):
+        currentBoro = currentElement.text()
+
+    # check for <strong> followed immediately by <p>
+    if currentElement.is_selector('strong') and nextElement.is_selector('p'):
+        rows.append([currentBoro, currentElement.text(), nextElement.text()])
+
+```
+
+This code yields a nice list of lists containing the borough, streets_closed, and description:
+
+```python
+
+[
+  ["borough", "streets_closed", "description"],
+  ["Manhattan", "East 13th Street between 2nd Avenue and 1st Avenue", "This street will be closed Saturday from 8 am to 6 pm and Sunday from 9 am to 6 pm through 10/3/21 to facilitate crane operation."],
+  ["Manhattan", "East 19th Street between Park Avenue South and Broadway", "This street will be closed Saturday and Sunday from 7 am to 7 pm through 10/3/21 to facilitate crane operation."],
+  ...
+]
+
+```
+
+### Step 4: Convert the data into a Starlark dataframe
+
+To commit this as the body of the dataset, we convert the list of lists to Starlark dataframe using `dataframe.Dataframe()`
+
+```
```

Comment:
add `python`

Reply:
Do you mean call it a "python DataFrame" instead of a "Starlark DataFrame"?